internal/fluentbit/ports/ports.go

```diff
@@ -0,0 +1,7 @@
+package ports
+
+const (
+	HTTP       = 2020
+	Metrics    = 2021
```

Comment:
This port should be for the directorySizeExporter. Maybe let's indicate that in a better way. I remember that we were unsure about the purpose last week.